pyfr/plugins/pseudostats.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from pyfr.mpiutil import get_comm_rank_root
+from pyfr.plugins.base import BasePlugin, init_csv
+
+
+class PseudoStatsPlugin(BasePlugin):
+    name = 'pseudostats'
+    systems = ['*']
+
+    def __init__(self, intg, cfgsect, prefix):
+        super().__init__(intg, cfgsect, prefix)
+
+        if intg.formulation != 'dual':
+            raise RuntimeError('Formulation {0} not supported by plugin {1}'
+                               .format(intg.formulation, self.name))
+
+        self.flushsteps = self.cfg.getint(self.cfgsect, 'flushsteps', 500)
+
+        self.count = 0
+        self.stats = []
+        self.tprev = intg.tcurr
+
+        fvars = ','.join(intg.system.elementscls.privarmap[self.ndims])
```

Comment:
I think this should be the conservative variable map as that is what the reduction is based on.